src/crypto/CHIPCryptoPALOpenSSL.cpp

```diff
@@ -812,26 +822,36 @@ void ClearSecretData(uint8_t * buf, uint32_t len)
         }                                                                                                                          \
     } while (0)
 
+typedef struct Spake2p_Context
+{
+    EC_GROUP * curve;
+    BN_CTX * bn_ctx;
+    const EVP_MD * md_info;
+} Spake2p_Context;
+
 CHIP_ERROR Spake2p_P256_SHA256_HKDF_HMAC::InitInternal(void)
 {
     CHIP_ERROR error  = CHIP_ERROR_INTERNAL;
     int error_openssl = 0;
 
-    context.curve   = NULL;
-    context.bn_ctx  = NULL;
-    context.md_info = NULL;
+    Spake2p_Context * context = (Spake2p_Context *) &mSpake2pContext;
+    static_assert(sizeof(mSpake2pContext) >= sizeof(Spake2p_Context), "Need more memory for Spake2p Context");
```

Comment:
Minor nit; however, if this file has no other C++11 dependencies, recommend using the portable static assertion in nlassert.h.